WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsInputDialogFragment.java

```diff
@@ -8,21 +8,20 @@
 import android.text.TextUtils;
 import android.text.TextWatcher;
 import android.view.LayoutInflater;
-import android.view.View;
-import android.widget.EditText;
-import android.widget.TextView;
 
 import androidx.annotation.Nullable;
 import androidx.appcompat.app.AlertDialog;
 import androidx.appcompat.view.ContextThemeWrapper;
 import androidx.fragment.app.DialogFragment;
 
 import com.google.android.material.dialog.MaterialAlertDialogBuilder;
-import com.google.android.material.textfield.TextInputLayout;
 
 import org.wordpress.android.R;
+import org.wordpress.android.databinding.PostSettingsInputDialogBinding;
 import org.wordpress.android.util.ActivityUtils;
 
+import java.util.Objects;
```

Comment:
Suggestion (💡): Consider removing this and its `requireNonNull(...)` usage to keep this PR a bit more focused on the view binding work. As an example, you didn't include it on [this other PR](https://github.com/wordpress-mobile/WordPress-Android/pull/20933/files#diff-d7016f4360f799e1e75be761f8c81c16f29e3c0899a9595e94eb8740e6a3c183R98) of yours, and I wonder why.

I suggest that because instead of doing so, I would rather do the following:

```java
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Editable text = binding.postSettingsInputDialogEditText.getText();
                if (mListener != null && text != null) {
                    mCurrentInput = text.toString();
                    mListener.onInputUpdated(mCurrentInput);
                }
            }
```

However, as you can see, this has the potential to affect the functionality of `onClick(...)` so I would rather be very sure before doing this change, just because `mCurrentInput` is now being assigned a value within the `if` statement.

As such, I suggest to keep this piece of logic as is, without the `requireNonNull(...)` trick, just to get the compiler to stop complaining, because, you would anyway get a NPE should `postSettingsInputDialogEditText` ends-up being null, thus (kind of) hiding the problem.

Reply:
Yes, you are correct. I am reverting the changes and leaving it as it is to keep refactor PR as minimal as possible. Will fix the lint issues in kotlinization PR.

Reply:
Awesome, thanks for that @neeldoshii ! 🥇 

FYI: Btw, I am not sure why you are doing that, but feel free to add a separate commit with any change that is suggested during a PR review. Instead, if you keep rebasing your single commit it creates a problem where I also have to rebase the draft PR for CI. Also, in general, after any PR gets open, the guidance is to not rebase and force-push any more, as this makes it difficult to review the changes, as a review ideally would have to be done from scratch. Just adding extra commit on top is perfectly fine. Does that make sense to you? 🙏